data/manual-published-branches.yaml

```
@@ -1,18 +1,19 @@
 version:
   published:
-    - '6.1 (Rapid Release)'
+    - '7.0 (Upcoming)'
+    - '6.3 (Rapid Release)'
     - '6.0 (Major Release)'
     - '5.0'
     - '4.4'
     - '4.2'
   active:
-    - '6.1'
+    - '6.3'
```

Should probably add 7.0 as an active version as well.

```suggestion
    - '7.0'
    - '6.3'
```

---

Done!